scripts/develop-module.js

```diff
@@ -0,0 +1,299 @@
+    const ds6StylesContent = ds4StylesContent;
+
+    fs.writeFileSync(path.join(baseFolderPath, `${newModuleData.id}.less`), baseStylesContent);
+    fs.writeFileSync(path.join(ds4FolderPath, `${newModuleData.id}.less`), ds4StylesContent);
+    fs.writeFileSync(path.join(ds6FolderPath, `${newModuleData.id}.less`), ds6StylesContent);
+}
+
+function addSourceBundleStyles(newModuleData) {
+    const ds4FileBundlePath = path.join(__dirname, '..', 'src', 'less', 'bundles', 'skin', 'ds4', 'skin.less');
+    const ds6FileBundlePath = path.join(__dirname, '..', 'src', 'less', 'bundles', 'skin', 'ds6', 'skin.less');
+
+    if (fs.readFileSync(ds4FileBundlePath, 'utf8').match(`${newModuleData.id}.less`)) {
+        log.warn('[STYLES][skin.less] Skin styles already exists!');
+        return;
+    }
+
+    fs.appendFileSync(ds4FileBundlePath, `@import "../../../${newModuleData.id}/ds4/${newModuleData.id}.less";\n`);
+    fs.appendFileSync(ds6FileBundlePath, `@import "../../../${newModuleData.id}/ds6/${newModuleData.id}.less";\n`);
+}
+
+function addPackagingStyles(newModuleData) {
+    addPackagingStyleDependencies(newModuleData);
+    addPackagingStylesDs4Reference(newModuleData);
+    addPackagingStylesDs6Reference(newModuleData);
+    addPackagingStylesIndexReference(newModuleData);
+}
+
+function addPackagingStyleDependencies(newModuleData) {
+    const dependenciesFile = 'index.browser.json';
+    const dependenciesContent = fs.readFileSync(dependenciesFile, 'utf8');
+    if (dependenciesContent.match(newModuleData.id)) {
+        log.warn('[STYLES][%s] Dependencies already exists!', dependenciesFile);
+        return;
+    }
+
+    const dependenciesLines = dependenciesContent.split('\n');
+    const finalContent = [];
+    let isDeps = false;
+    for (const line of dependenciesLines) {
+        if (line.match(/"dependencies"\s*\t*:\s*\t*\[/)) {
+            isDeps = true;
+        } else if (isDeps && line.match(']')) {
+            isDeps = false;
+            finalContent.push(`${finalContent.pop()},`);
+            finalContent.push(`        "./${newModuleData.id}"`);
+        }
+
+        finalContent.push(line);
+    }
+
+    fs.writeFileSync(dependenciesFile, finalContent.join('\n'));
+}
+
+function addPackagingStylesDs4Reference(newModuleData) {
+    const ds4File = `${newModuleData.id}[ds-4].less`;
+    if (fs.existsSync(ds4File)) {
+        log.warn('[STYLES][%s] DS4 root styles already exists!', ds4File);
+        return;
+    }
+
+    fs.writeFileSync(ds4File, `@import "./dist/${newModuleData.id}/ds4/${newModuleData.id}.css";\n`);
+}
+
+function addPackagingStylesDs6Reference(newModuleData) {
+    const ds6File = `${newModuleData.id}.less`;
+    if (fs.existsSync(ds6File)) {
+        log.warn('[STYLES][%s] DS6 root styles already exists!', ds6File);
+        return;
+    }
+
+    fs.writeFileSync(ds6File, `@import "./dist/${newModuleData.id}/ds6/${newModuleData.id}.css";\n`);
+}
```

Comment:
We could again consolidate this into a single method. This seems exactly the same exept for ds4/ds6

Just pass in the dsVersion as an argument. So either ds4 or ds6. Then you can do 
const dsFile = `${newModuleData.id}${dsVer === 'ds4' ? [ds-4] : ''}.less`; 
Then the rest just substitute `dsVersion`